pep-0688.rst

```diff
@@ -0,0 +1,252 @@
+PEP: 688
+Title: Making the buffer protocol accessible in Python
+Author: Jelle Zijlstra <jelle.zijlstra@gmail.com>
+Sponsor: Jelle Zijlstra <jelle.zijlstra@gmail.com>
+Status: Draft
+Type: Standards Track
+Content-Type: text/x-rst
+Created: 23-Apr-2022
+Python-Version: 3.12
+
+
+Abstract
+========
+
+This PEP proposes a mechanism to inspect in Python whether a type implements
```

Comment:
Phrasing suggestion
```suggestion
This PEP proposes a mechanism within Python to inspect if a type implements
```

Reply:
Thanks, but I prefer the current wording here

Reply:
For what its worth, I also find the current phrasing somewhat hard to parse and understand, particularly so when it is the very first sentence of the PEP's abstract and thus there isn't much in the way of context clues to easily infer the intended interpretation. What about these other options that also address the issue?

```suggestion
This PEP proposes a mechanism for Python code to inspect whether a type implements
```

```suggestion
This PEP proposes a Python-level mechanism to inspect whether a type implements
```

```suggestion
This PEP proposes a mechanism to inspect at the Python level whether a type implements
```

```suggestion
This PEP proposes a mechanism in Python to inspect whether a type implements
```

Reply:
Thanks, I like the first suggestion here. Just pushed that change.